tardis/io/model_reader.py

```diff
@@ -578,7 +578,7 @@ def transport_to_dict(transport):
         "photo_ion_estimator_statistics": transport.photo_ion_estimator_statistics,
         "r_inner": transport.r_inner_cgs,
         "r_outer": transport.r_outer_cgs,
-        "seed": transport.seed,
```

Comment:
Change this back might resolve the issue

Reply:
I have resolved these test cases. Have stored the base_seed of the packet_source in the dict and HDF file.